packages/typir/src/features/conversion.ts

```diff
@@ -180,11 +176,47 @@ export class DefaultTypeConversion implements TypeConversion {
         return 'NONE';
     }
 
-    protected isTransitivelyConvertable(_from: Type, _to: Type, _mode: ConversionModeForSpecification): boolean {
-        // TODO calculate transitive relationship
+    protected existsEdgePath(from: Type, to: Type, mode: ConversionModeForSpecification): boolean {
+        const visited: Set<Type> = new Set();
+        const stack: Type[] = [from];
+
+        while (stack.length > 0) {
```

Comment:
Thanks for the iterative implementation!

Reply:
I hope it helps with performance.